packages/matchbox/src/components/Toggle/styles.js

```diff
@@ -0,0 +1,68 @@
+import { tokens } from '@sparkpost/design-tokens';
+import styled from 'styled-components';
+
+export const toggle = props => `
+  display: inline-block;
+  position: relative;
+  height: 1.25rem;
+  width: 2.25rem;
+
+  opacity: ${props.disabled ? '0.6' : '1'}
+
+  &:hover {
+    cursor: ${props.disabled ? 'not-allowed' : 'pointer'};
+  }
+`;
+
+export const indicator = props => `
+  pointer-events: none;
+  position: absolute;
+  top: 1px;
+  left: 1px;
+  bottom: 1px;
+  width: calc(2.25rem / 2);
```

Comment:
This width value could maybe be stored as a variable to make managing this more clear?